aineko/core/dataset.py

```diff
@@ -22,7 +22,7 @@
 import json
 from typing import Any, Dict, Optional
 
-from confluent_kafka import Consumer, Message, Producer
+from confluent_kafka import Consumer, Message, Producer  # type: ignore
```

Comment:
```suggestion
from confluent_kafka import Consumer, Message, Producer
```
See previous comment about overrides.